docs/hazmat/backends/openssl.rst

```diff
@@ -125,6 +125,62 @@ seeded from the same pool as ``/dev/random``.
 +------------------------------------------+------------------------------+
 
 
+FIPS mode
+---------
+
+Federal Information Processing Standards is a public standard for use by
+US American government agencies. OpenSSL in FIPS mode prohibits the use of
+non-approved algorithms and parameters.
+
+Key restrictions
+~~~~~~~~~~~~~~~~
+
+* _`Diffie-Hellman key exchange` requires key size equal or greater than
+  2048 bits.
+* _`DSA` requires key size equal or greater than 2048 bits.
+* _`RSA`  requires key size equal or greater than 2048 bits and the
+  public exponent must be 65537.
+
+Prohibited message digests
```

Comment:
Wouldn't this be easier if we listed _approved_ algorithms, not banned ones?

FIPS is expressed as an approval list, so this is honestly confusing.